shared/middleware/files_api.rb

```diff
@@ -201,6 +205,13 @@ def get_file(endpoint, encrypted_channel_id, filename, code_projects_domain_root
     result[:body]
   end
 
+  # A list of some file types that are safe to view in the browser without
+  # risking script execution. Initially limited to images since it is useful
+  # to view these via the browser context menu.
+  def safely_viewable_file_type?(extension)
+    %w(.jpg .jpeg .gif .png).include? extension
```

Comment:
It looks like there might be cases where we haven't standardized the extension case before we pass it into this helper.  Can we either downcase here or write a test to verify that we always downcase the extension first?